incorrect filenames in docker-compose/Makefile

Got error below when running `make add-catalog`:

```
/bin/sh: /applications/openwhisk/openwhisk-devtools/docker-compose/openwhisk-catalog/packages/installCatalogUsingWskDeploy.sh: No such file or directory
make: *** [init-catalog] Error 127
```

It happens because the actual file name in `openwhisk-catalog/packages` is `installCatalogUsingWskdeploy.sh`, not `installCatalogUsingWskDeploy.sh`. Letter `D` is lowercase.

Also, when running:

```
make create-provider-alarms
make create-provider-kafka
make create-provider-cloudant
```

Name of the related script should be `installCatalog.sh`, not `installCatalogUsingWskDeploy.sh`.

--- a/docker-compose/Makefile
+++ b/docker-compose/Makefile
@@ -15,6 +15,7 @@
 # limitations under the License.
 #
 
+SHELL:=/bin/bash
 UNAME_STR ?= $(shell uname)
 
 # detect local ip of host as this is needed within containers to find the OpenWhisk API container

--- a/docker-compose/Makefile
+++ b/docker-compose/Makefile
@@ -304,7 +304,7 @@ wait-for-docker-compose-up:
 	echo "Waiting till all containers are powered up... ";
 	sleep 1;
 	@trycount=0; \
-	trycounttimeout=30; \
+	trycounttimeout=60; \
 	up=`$(shell cat $(TMP_HOME)/tmp/openwhisk/local.env) docker-compose --project-name openwhisk ps | grep Up -c`; \
 	servicescount=`$(shell cat $(TMP_HOME)/tmp/openwhisk/local.env) docker-compose -f $(DOCKER_COMPOSE_FILE) config --services | wc -l | xargs`; \
 	docker_ps() { \

--- a/docker-compose/Makefile
+++ b/docker-compose/Makefile
@@ -485,7 +485,7 @@ $(addprefix download-package-,$(PACKAGES)):
 	echo "Downloading package" $(PACKAGE_NAME) "into" $(PACKAGE_HOME)
 	rm -rf ./openwhisk-package-$(PACKAGE_NAME)*
 	if [ "$(PACKAGE_HOME)" = "./openwhisk-package-$(PACKAGE_NAME)" ]; then \
-	    curl -s -O ./openwhisk-package-$(PACKAGE_NAME).tar.gz -L https://api.github.com/repos/apache/openwhisk-package-$(PACKAGE_NAME)/tarball/master > ./openwhisk-package-$(PACKAGE_NAME).tar.gz; \
+	    curl -s --output ./openwhisk-package-$(PACKAGE_NAME).tar.gz -L https://api.github.com/repos/apache/openwhisk-package-$(PACKAGE_NAME)/tarball/master; \
 	    mkdir openwhisk-package-$(PACKAGE_NAME); \
 	    tar -xf ./openwhisk-package-$(PACKAGE_NAME).tar.gz --strip 1 -C openwhisk-package-$(PACKAGE_NAME); \
 	else \
@@ -497,7 +497,14 @@ $(addprefix install-package-,$(PACKAGES)):
 	$(eval PACKAGE_NAME:= $(shell echo $(@) | cut -b 17-))
 	$(eval PACKAGE_HOME := $(PACKAGE_$(shell echo $(PACKAGE_NAME) |  tr 'a-z' 'A-Z')_HOME))
 	cd $(PACKAGE_HOME) && \
-	$(shell cat $(TMP_HOME)/tmp/openwhisk/providers.env) ./installCatalogUsingWskdeploy.sh $(realpath $(OPENWHISK_PROJECT_HOME))/ansible/files/auth.whisk.system $(DOCKER_HOST_IP) "http://$(DOCKER_HOST_IP):5984" $(OPEN_WHISK_DB_PREFIX) $(DOCKER_HOST_IP)
+	$(shell cat $(TMP_HOME)/tmp/openwhisk/providers.env) ./installCatalog.sh $(realpath $(OPENWHISK_PROJECT_HOME))/ansible/files/auth.whisk.system $(DOCKER_HOST_IP) "http://$(DOCKER_HOST_IP):5984" $(OPEN_WHISK_DB_PREFIX) $(DOCKER_HOST_IP)
+
+### Separate target for alarms because the installation requires an additional parameter
+install-package-alarms:
+	$(eval PACKAGE_NAME:= $(shell echo $(@) | cut -b 17-))
+	$(eval PACKAGE_HOME := $(PACKAGE_$(shell echo $(PACKAGE_NAME) |  tr 'a-z' 'A-Z')_HOME))
+	cd $(PACKAGE_HOME) && \
+	$(shell cat $(TMP_HOME)/tmp/openwhisk/providers.env) ./installCatalog.sh $(realpath $(OPENWHISK_PROJECT_HOME))/ansible/files/auth.whisk.system $(DOCKER_HOST_IP) $(DOCKER_HOST_IP) "http://$(DOCKER_HOST_IP):5984" $(OPEN_WHISK_DB_PREFIX) $(DOCKER_HOST_IP) \
 
 .PHONY: $(addprefix start-provider-,$(PACKAGES))
 $(addprefix start-provider-,$(PACKAGES)):

--- a/docker-compose/Makefile
+++ b/docker-compose/Makefile
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-#SHELL:=/bin/bash
+SHELL:=/bin/bash
 UNAME_STR ?= $(shell uname)
 
 # detect local ip of host as this is needed within containers to find the OpenWhisk API container

--- a/docker-compose/Makefile
+++ b/docker-compose/Makefile
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-SHELL:=/bin/bash
+#SHELL:=/bin/bash
 UNAME_STR ?= $(shell uname)
 
 # detect local ip of host as this is needed within containers to find the OpenWhisk API container

--- a/docker-compose/Makefile
+++ b/docker-compose/Makefile
@@ -88,7 +88,7 @@ download-src:
 download-catalog:
 	if [ "$(OPENWHISK_CATALOG_HOME)" = "$(CURDIR)/openwhisk-catalog" ]; then \
 	    rm -rf $(CURDIR)/openwhisk-catalog*; \
-	    curl -s -O ./openwhisk-catalog.tar.gz -L https://api.github.com/repos/apache/openwhisk-catalog/tarball/master > ./openwhisk-catalog.tar.gz; \
+	    curl -s --output ./openwhisk-catalog.tar.gz -L https://api.github.com/repos/apache/openwhisk-catalog/tarball/master; \
 	    mkdir openwhisk-catalog; \
 	    tar -xf ./openwhisk-catalog.tar.gz --strip 1 -C openwhisk-catalog; \
 	else \

